app/scripts/services/membership/membership.js

```diff
@@ -0,0 +1,172 @@
+'use strict';
+
+angular
+  .module('openshiftConsole')
+  .factory('MembershipService', function($filter) {
+
+    var annotation = $filter('annotation');
+
+    var userRoles = function(roleBindings, user) {
+                      return _.map(
```

Comment:
im not a fan of this indention style, userRoles is just a function, indent the function content once